src/language.h

```diff
@@ -107,7 +111,23 @@ extern const LanguageMetadata *_current_language;
 extern std::unique_ptr<icu::Collator> _current_collator;
 #endif /* WITH_ICU_I18N */
 
+/** The number digits available in a uint64_t. */
+constexpr int DIGITS_IN_UINT64_T = 20;
+/** Table with the text to place between each of the digits of a number. */
```

Comment:
This comment could be more descriptive:
* Are the digits counted from left to right, or from right to left?
* Are the strings inserted before or after the dight?

Something like ``Text at index "20 - i" is inserted after the digit with value "10**i".`` Though I have no idea whether this is correct, or the other way around. Hence why I am asking for a better comment :)